package.nls.json

```diff
@@ -6,6 +6,7 @@
   "contributes.commands.java.view.package.refresh": "Refresh",
   "contributes.commands.java.project.build.workspace": "Build Workspace",
   "contributes.commands.java.project.clean.workspace": "Clean Workspace",
+  "contributes.commands.java.view.package.revealInProjectExplorer": "Open Java Projects",
```

Comment:
`revealInProjectExplorer` vs "Open Java Projects"? should it be something like "Reveal in Java Projects"?

Reply:
+1

Reply:
Updated.